### DISCUSSION TITLE
Trojan Detected

### DISCUSSION THREAD

**Discussion by Hissss-Hea-I-Come** [Security-Onion-Solutions/securityonion#11294] [Unsupported Versions] [answered]
Hi there, just downloaded 2.4.10-20230821 ISO image from https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO: checksum is accurate but my bitdefender anti virus detected "Trojan" and same resulted with microsoft defender and Sophos. Can this be a false positive...has anyone else experienced this? 

Thanks

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.4/download.html:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/83c42e4d-ecb5-4dd1-ab91-752a4915e5e6)

**Comment by Hissss-Hea-I-Come** [comment]
Thanks for the info. Doubts cleared. I appreciate your assistance.
